build/assets/scripts/high-performance-hooks.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+pid=$(jq '.pid' /dev/stdin)
+mask=$1
+ns=$(ip netns identify "${pid}")
+
+[[ -n "${pid}" ]] && [[ -n "${mask}" ]] && [[ -n "${ns}" ]] || exit 0
+
+ip netns exec "${ns}" mount -o remount,rw /sys || exit 0
+ip netns exec "${ns}" find /sys/devices -type f -name rps_cpus -exec sh -c "echo ${mask} | cat > {}" \;
```

Comment:
any reason to use a different find here with respect to https://github.com/openshift-kni/performance-addon-operators/pull/450/commits/bde1fc3aa43f0b62e1c72259d47ab9c6d24b663f#diff-b550247b819f015b31bacae7c601e2ef1045d94b6ff48ecb29c3cb362578be75R306 ?

Reply:
funny thing, find doesn't find the rps_cpus files  in /sys/class/net/ dir.
It does find them in  /sys/class/net/vethxyz/ or in /sys/devices/

Reply:
is something about an implementation difference of the sys/class vs /sys/devices

Reply:
ok, no problem. Worth a comment too. Others (or our future selves) may well wonder the same thing in the future.

Reply:
Will add a comment in the script, thanks